index.html

```diff
@@ -0,0 +1,172 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="assets/utiles.css">
+    <title>Event Website</title>
+</head>
+
+<body>
+    <header class="header">
+
+        <!-- Desktop menu -->
+        <div class="header-top">
+            <ul>
+                <li><a href=""><img src="assets/icones/1.png" alt="github" width="30" height="30"></a></li>
+                <li><a href=""><img src="assets/icones/4.png" alt="tweeter" width="30" height="30"></a></li>
+                <li><a href="">Login</a></li>
+                <li><a href="">Sign Up</a></li>
+
+            </ul>
+
+        </div>
+        <div class="desktop-menu">
+            <a href="index.html"><img src="assets/images/GE.png" alt="" width="120" class="logo"></a>
+            <nav>
+                <ul class="desktop-nav">
+                    <li><a href="about.html">About</a></li>
+                    <li><a href="">Program</a></li>
+                    <li><a href="">Join</a></li>
+                    <li><a href="">Support</a></li>
+                    <li><a href="">News</a></li>
+                    <li class="ge-campaigne"><a href="">GE campaign</a></li>
+
+                </ul>
+            </nav>
+
+        </div>
+        <button class="menu-button">
+            <img src="assets/icones/menu.png" alt="menu" width="45" height="45">
+        </button>
```

Comment:
### Your Burger Icon
![nav](https://user-images.githubusercontent.com/93039359/211143758-f16c71a4-ee13-4da1-a1d3-076e9123a5fe.PNG)

### Design Burger Icon
![nav](https://user-images.githubusercontent.com/93039359/211143766-f4d91033-3fc3-4833-89ab-8abfd8445c67.PNG)

- [ ] You can see that in the design burger icon has a background but you don't have it. Kindly follow the design and add a background to this section 😊